src/s2wasm.h

```diff
@@ -1248,6 +1249,22 @@ class S2WasmBuilder {
     wasm.addExport(exp);
   }
 
+  void getDyncallThunk(const std::string& sig) {
+    // hard-code the sig for testing for now. vi is the signature for static destructors
+    assert(sig == "vi");
+    auto* targetType = ensureFunctionType(sig, &wasm, wasm.allocator);
+    wasm::Builder wasmBuilder(wasm);
+    std::vector<NameType> params {{"$0", i32}, {"$1", i32}};
+    std::vector<NameType> locals;
+    Function* f = wasmBuilder.makeFunction(std::string("dynCall_") + sig, std::move(params), none, std::move(locals));
```

Comment:
very minor comment: would it be possible to write `{}` for the locals argument? would be nice to avoid the extra line to define them before.